wagtail/wagtailembeds/embeds.py

```diff
@@ -15,10 +15,13 @@
 from wagtail.wagtailembeds.models import Embed
 
 
-class EmbedNotFoundException(Exception):
+class EmbedException(Exception):
```

Comment:
Is it wise to be introducing a new exception class in a PR being backported to 0.8?